docs/index.md

```diff
@@ -16,9 +18,9 @@ Please review [system requirements](system-requirements.md) before proceeding wi
 ## Setup
 
 1. [Docksal environment setup](env-setup.md)
-    
-    This is done **one time per host** and should be performed by everyone.
+
```

Comment:
It's an issue with the numbering again. I added some custom css that will hopefully fix that.